test/NuGet.Clients.Tests/NuGet.VisualStudio.Test/IVsPathContextProviderTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Moq;
+using Xunit;
+
+namespace NuGet.VisualStudio.Test
+{
+    public class IVsPathContextProviderTests
+    {
+        // Verify interface method can be used across assembly boundaries
+        [Fact]
+        public void TryCreateContext_EmbeddableTest()
+        {
+            var provider = Mock.Of<IVsPathContextProvider>();
```

Comment:
From [the NuGet team coding guidelines](https://github.com/NuGet/Home/wiki/Coding-Guidelines#unit-test-structure):
> The contents of every unit test should be split into three distinct stages, optionally separated by these comments

The keyword here is *optionally*.